OrganizationVcf.md

```diff
@@ -0,0 +1,75 @@
+# Organization Contacts in VCard format
```

Comment:
I think this needs much more info to be useful. What contacts, how do you encode them?